corehq/warehouse/transforms/sql/app_status_form_staging.sql

```diff
@@ -47,7 +47,7 @@ SELECT
 	ON app_dim.id = app_status.app_dim_id and user_dim.id = app_status.user_dim_id
 	WHERE form_staging.user_id <> '' and user_dim.doc_type='CommCareUser'
         WINDOW wnd AS (
-            PARTITION BY form_staging.user_id, form_staging.app_id ORDER BY received_on AT TIME ZONE 'UTC' DESC
+            PARTITION BY user_dim.id, COALESCE(app_dim.id, '-1') ORDER BY received_on AT TIME ZONE 'UTC' DESC
```

Comment:
why is the `-1` a string here and an int in `app_status_fact.sql`?

Reply:
It was initially a string because i was coalescing with `app_id` and the both arguments had to be of the same type. It doesnt really matter because its just a dummy value for the partition so there are no nulls (which get treated oddly). It doesnt get saved anywhere here, but I can change it to an integer for the sake of consistency